content/projects/tdd/password-checker/part1/_index.md

```diff
@@ -149,4 +149,12 @@ Please don't re-implement the same check in two different places. Good code is D
 
 ## Java Resources
 
-- https://howtodoinjava.com/junit5/expected-exception-example/
\ No newline at end of file
+- https://howtodoinjava.com/junit5/expected-exception-example/
+
+## Instructions for reviewers
+
+1. **Conditions** - There are 7 conditions above, so make sure that all conditions are being used for both of the functions passwordIsValid and PasswordStrength. 
```

Comment:
Is this in the projects instructions above the review instructions should be DRY, if we are rewording the above instructions it is not dry